coderd/audit/request.go

```diff
@@ -43,6 +44,9 @@ func ResourceTarget[T Auditable](tgt T) string {
 		return typed.Username
 	case database.Workspace:
 		return typed.Name
+	case database.WorkspaceBuild:
+		// this isn't used
```

Comment:
Does the resource target not show anywhere in the audit log?

Reply:
I see now it does indeed not show up anywhere (except the database I assume).

Reply:
Yeah, we decided not to have a diff for this particular type of log; however, we will definitely keep our ears to the ground to see if people voice concern. When we start to audit through the CLI, I see us surfacing more info.

Reply:
That makes sense!